src/applications/claims-status/components/claim-status-tab/RecentActivity.jsx

```diff
@@ -193,10 +193,11 @@ export default function RecentActivity({ claim }) {
     );
   };
   const requestType = itemStatus => {
-    if (itemStatus === 'NEEDED_FROM_OTHERS') {
+    if (itemStatus === 'NEEDED_FROM_OTHERS' && !cstFriendlyEvidenceRequests) {
       return 'Request for others';
     }
-    return 'Request for you';
+    if (itemStatus === 'NEEDED_FROM_YOU') return 'Request for you';
+    return undefined;
```

Comment:
I don't think you technically need this but it does not really hurt anything to be present